model/change_point_detection.go

```diff
@@ -428,3 +428,60 @@ func createChangePoint(ctx context.Context, env cedar.Environment, resultToUpdat
 	_, err := env.GetDB().Collection(perfResultCollection).UpdateOne(ctx, filter, update)
 	return errors.Wrap(err, "Unable to create change point")
 }
+
+type ChangePointStub struct {
+	PerfResultID string `json:"perf_result_id"`
+	Measurement  string `json:"measurement"`
+}
+
+func TriageChangePoints(ctx context.Context, env cedar.Environment, changePoints []ChangePointStub, status TriageStatus) error {
+	coll := env.GetDB().Collection(perfResultCollection)
+
+	var conditions []bson.M
+	for _, stub := range changePoints {
+		conditions = append(conditions, bson.M{
+			perfIDKey: stub.PerfResultID,
+			bsonutil.GetDottedKeyName(perfAnalysisKey, perfAnalysisChangePointsKey, perfChangePointMeasurementKey): stub.Measurement,
+		})
+	}
+	filter := bson.M{
+		"$or": conditions,
+	}
+
+	cur, err := coll.Find(ctx, filter)
+	if err != nil {
+		return errors.Wrap(err, "Could not execute query finding change points for triage")
+	}
+	var results []PerformanceResult
+	if err := cur.All(ctx, &results); err != nil {
+		return errors.Wrap(err, "Could not decode performance results for triage")
+	}
+
+	for _, stub := range changePoints {
+		for _, res := range results {
+			if res.ID == stub.PerfResultID {
+				for _, cp := range res.Analysis.ChangePoints {
+					if cp.Measurement == stub.Measurement {
+						goto cont
+					}
+				}
+			}
+		}
+		return errors.Errorf("Could not find change point <%s> for performance result %s", stub.Measurement, stub.PerfResultID)
+	cont:
+		continue
+	}
```

Comment:
Something like this is needed since performance results can have multiple change points being triaged at once.